client/state/notification-settings/actions.js

```diff
@@ -78,13 +59,13 @@ function buildSavePayload( source, settings ) {
 	}
 }
 
-export const showSaveSuccessNotice = () =>
```

Comment:
These aren't used outside the file. Removing the `export` should make it clearer when they're safe to remove we revise fetching and saving behaviors, since ESLint will flag them as unused.

It'd probably be even better to just inline the behavior where they're used below, since they're only used once. But I made this change instead to keep the diff minimal.